Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -249,7 +249,15 @@ OSD locations defined in the CRUSH map and topology labels on nodes.
 Refer to the [krbd-options](https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options)
 for more details.
 
-Execute the following steps:
+Execute the following step to enable read affinity for a specific ceph cluster:
+
+* Patch the ceph cluster CR to enable read affinity:
+
```

Comment:
How about also showing the yaml snippet for readability in case they prefer to edit their manifests? 
```yaml
  csi:
    readAffinity:
      enabled: true
```